sdk/tables/data-tables/src/utils/accountConnectionString.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TableServiceClientOptions } from "..";
+import { ConnectionString } from "./connectionString";
+import { TablesSharedKeyCredential } from "../TablesSharedKeyCredential";
+import { createPipelineFromOptions } from "@azure/core-http";
+
+/**
+ * Gets client parameters from an Account Connection String
+ * Only supported in Node.js not supported for Browsers
+ * @param extractedCreds parsed connection string
+ * @param options TablesServiceClient options
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Just wondering, could we add an explicit return type here by importing ClientParamsFromConnectionString from `./connectionString` and then using that type, or was there some reason we decided not to do that?

Reply:
Absolutely! I don't remember why I ignored it, it may have been to avoid the circular dependency. Anyway I also moved this interface to internalModels and now setting the return type